Microsoft.Toolkit.Uwp.UI.Media/Geometry/CultureShield.cs

```diff
@@ -0,0 +1,61 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Globalization;
+
+namespace Microsoft.Toolkit.Uwp.UI.Media.Geometry.Common
+{
+    /// <summary>
+    /// Class which can be used to encapsulate code statement(s)
+    /// so that they are executed in a specific culture.
+    /// <para />
+    /// Usage example:
+    /// <para />
+    /// The following code block will be executed using the French culture.
+    /// <para />
+    /// using (new CultureShield("fr-FR"))
+    /// <para />
+    /// {
+    /// <para />
+    ///    ...
+    /// <para />
+    /// }
+    /// </summary>
+    public class CultureShield : IDisposable
```

Comment:
Should this be public? Do we want to expose this?

Reply:
Agreed that we should keep this type internal, since it's essentially an implementation detail.
Even if we wanted to make it public, then this would be the wrong package to expose it.
I'd just make this internal here 🙂

Reply:
Converted it into `internal ref struct`.